appeals/web/src/server/lib/nunjucks-template-builders/gds-components-properties-jsdoc.js

```diff
@@ -717,3 +717,19 @@
  * @property {{value: number}} hour
  * @property {{value: number}} minute
  */
+
+/**
+ * @typedef CaseNotesProperties
+ * @type {Object<string, any>}
+ * @property {CaseNote[]} casenotes
+ */
+
+/**
+ * @typedef CaseNote
+ * @type {Object<string, any>}
+ * @property {string} date
```

Comment:
These property names could be a little more descriptive. Suggest:

- `days` (I think this should be `day`? ) to `dayOfWeek`
- `user` to `userName`
- `comment` to `commentText`